src/building-blocks/FormItem.tsx

```diff
@@ -69,18 +69,27 @@ class FormItem extends Component<IFormFieldProps> {
     };
   }
 
+  private get formItemExtra () {
+    const { fieldConfig } = this.props
+      , { formItemProps } = fieldConfig;
+    return (
+      <span>
+        { formItemProps && formItemProps.extraFieldConfig && renderLabel(formItemProps.extraFieldConfig) }
```

Comment:
I would prefer this logic to be split out (e.g. return null if the first two aren't truthy)